analyzer/pkg/rpm/rpm.go

```diff
@@ -62,21 +67,63 @@ func (a rpmPkgAnalyzer) parsePkgInfo(packageBytes []byte) (pkgs []types.Package,
 	for _, pkg := range pkgList {
 		arch := pkg.Arch
 		if arch == "" {
-			arch = "(none)"
+			arch = "None"
 		}
+
+		// parse source rpm
+		var srcName, srcVer, srcRel string
+		if pkg.SourceRpm != "(none)" && pkg.SourceRpm != "" {
+			// source epoch is not included in SOURCERPM
+			srcName, srcVer, srcRel, _, _ = splitFileName(pkg.SourceRpm)
```

Comment:
Since `epoch` is dropped do we still need this logic https://github.com/aquasecurity/fanal/pull/142/files#diff-a25c031570c1f818ffca9c3824028e5731f708802a6ff278b1e6020747bab04cR116-R125

Reply:
`splitFilename` is ported from python, so it is a generic function. In this case, `SourceRpm` doesn't include an epoch and we have to use the epoch of the binary package.

Reply:
I understand your point. But I meant to say that we return epoch here https://github.com/aquasecurity/fanal/blob/15afdbeacb4bc40e101d9ef12ea1a7ab483ad7ce/analyzer/pkg/rpm/rpm.go#L124 

So in our case we don't use the return value could we not remove this logic: https://github.com/aquasecurity/fanal/blob/15afdbeacb4bc40e101d9ef12ea1a7ab483ad7ce/analyzer/pkg/rpm/rpm.go#L117-L121

My apologies if I missed something obvious! 🙇🏻‍♂️

Reply:
You're totally right. We can remove it. I just wanted to have the same functionality as the original function in Python. If they are the same logic between Python and Go, we can easily debug the Go version when it has a bug.
On the other hand, I understand what you mean. It is unnecessary and might be confusing. I've removed it.
https://github.com/aquasecurity/fanal/pull/142/commits/c76ea78da0c290a0ea9d7395421b5df8f551f437